### DISCUSSION TITLE
Bring back the best features! Please!!

### DISCUSSION THREAD

**Discussion by SGTspicy** [Courseplay/Courseplay_FS22#3479] [Ideas]
Can we Please bring back the fs19 features where you could auto detect the edge of the field and record a custom field course? Not to toot my own horn but I can and have come up with courses that are way faster and more efficient than any courseplay autogenerated course but I have no way to record them anymore unless I want to move 50,000 waypoints individually... which I don't... is there a legit reason the two BEST features were done away with from fs19 to fs22? and will these two features be coming back in fs25?

**Comment by Tensuko** [comment]
Won't happen.

**Reply by SGTspicy** [reply]
is there any reason why it was taken out and not coming back?

**Reply by Tensuko** [reply]
Totaly unnecessary feature that we would have to maintain + new troubles that come up.
Use the generator and get to know how to use it, then you don't have to "move 50.000 Waypoints"
